docs/modules/release-notes/pages/5-3-0.adoc

```diff
@@ -0,0 +1,231 @@
+https://github.com/hazelcast/hazelcast/pull/23883[#23883]
+* When advanced networking is enabled, the Kubernetes discovery plugin might have been discovering several endpoints (per each port)
+for each member's pod. The discovery plugin now matches only the private IP per endpoint, ignoring the port values.
+https://github.com/hazelcast/hazelcast/pull/23766[#23766]
+* Added support of link:https://aws.amazon.com/about-aws/whats-new/2022/10/amazon-machine-images-support-instance-metadata-service-version-2-default/#:~:text=Posted%20On%3A%20Oct%203%2C%202022,depth%20against%20unauthorized%20metadata%20access.[IMDSv2^] for Hazelcast's AWS Discovery plugin.
+https://github.com/hazelcast/hazelcast/pull/23545[#23545]
+* Added the support of discovering ECS and EC2 members on AWS. See xref:deploy:deploying-on-aws.adoc[Deploying a Cluster on Amazon AWS].
+https://github.com/hazelcast/hazelcast/pull/22411[#22411]
+* [.enterprise]*Enterprise* Enhanced the experience of using Hazelcast with persistence under Kubernetes by removing requirements for manual interventions. Previously, when there is a restart on such a cluster, the cluster states were needed to be changed manually. For some cases, this might have led to data loss. Now, the shutdown process is automated. See xref:kubernetes:kubernetes-auto-discovery.adoc#running-hazelcast-enterprise-with-persistence-under-kubernetes[Configuring Kubernetes] for details.
+https://github.com/hazelcast/hazelcast/pull/21844[#21844]
+
+=== Storage
+
+* Disk tier option of Tiered Storage is now disabled by default.
+https://github.com/hazelcast/hazelcast/pull/23747[#23747]
+
+=== Distribution
+
+* Hazelcast no longer depends on JRuby; the JRuby `ScriptFactory` dependency must be explicitly added to the application.
+https://github.com/hazelcast/hazelcast/pull/23355[#23355]
+* Added the Kafka Connect extension to the distribution.
+https://github.com/hazelcast/hazelcast/pull/23312[#23312]
+* Shaded dependencies for Hazelcast Platform have been combined in a dedicated package (`com.hazelcast.shaded`).
+https://github.com/hazelcast/hazelcast/pull/23124[#23124]
+
+=== Networking
+
+* Added socket options for per-socket keep-alive configuration: `keep-count`, `keep-idle-seconds`, and `keep-interval-seconds`.
+You can set these options using either the advanced network configurations or Hazelcast system properties.
+See xref:clusters:network-configuration.adoc#configuring-tcp-keep-alive[Configuring TCP Keep-Alive].
+https://github.com/hazelcast/hazelcast/pull/23816[#23816], https://github.com/hazelcast/hazelcast/pull/24050[#24050]
+
+=== SQL Engine
+
+* The `CREATE MAPPING` statement for Compact Serialization now supports nullable primitives.
+https://github.com/hazelcast/hazelcast/pull/24300[#24300]
+* Implemented the `GET_DDL` function.
+https://github.com/hazelcast/hazelcast/pull/23595[#23595]
+* Implemented the `SHOW DATA CONNECTIONS` statement to show the connections created by the `CREATE CONNECTIONS` statement.
```

Comment:
entire data connection management in SQL was implemented as part of 5.3, there are also other command like `SHOW RESOURCES`